### ISSUE TITLE
Premature end of file / HTTPS problems

### ISSUE THREAD

**Issue by nc-cbm** [bjornvester/xjc-gradle-plugin#27]
When trying to XJC the following XSD https://docs.oasis-open.org/ebxml-msg/ebms/v3.0/core/os/ebms-header-3_0-200704.xsd I get his error:

Caught fatal error exception from XJC
org.xml.sax.SAXParseException; systemId: http://www.w3.org/2001/xml.xsd; lineNumber: 1; columnNumber: 1; Premature end of file.
Premature end of file.

Opening the link http://www.w3.org/2001/xml.xsd takes me to https://www.w3.org/2001/xml.xsd, and changing the schema location resolves the error.
However it won't fully compile as I suspect some underlying XSDs use the same reference.

In short I suspect the package has problems handling http links that should be opened in https.
